### DISCUSSION TITLE
Tokens Studio Sets for Material Theme

### DISCUSSION THREAD

**Discussion by lmcnally47** [primefaces/community#3263] [Figma UI Kit] [answered]
I recently purchased the Figma UI kit.

I see that you can customize the set of design tokens using Tokens Studio, and the Aura theme tokens are set by default.

Is there a plan on the road map to add the material theme to the available design token sets in the primeOne figma UI kit?

If I wanted to configure the figma UI kit to use the material theme myself, would it be possible to overwrite the values/variables in the Aura design token preset in Tokens Studio with the theme values set in the javascript package? (primitive, semantic, and component tokens from this repo:
https://github.com/primefaces/primevue/blob/master/packages/themes/src/presets/material/base/index.js)

Just trying to decide if its worth going down this path or if its better to wait for an update to the UI kit before using the material theme.
 
Thanks!

**Comment by cagataycivici** [comment, marked as answer]
No plans to offer Material version as it is mostly implemented mostly with custom CSS, it is an extended theme, not part of PrimeOne directly. Meaning, a material token set will not be the 100% same as in code.

Lara and Nora on the other hand can be implemented in Figma UI Kit since they are fully token based, no custom CSS.

We'll focus on improving Aura as our foundation at the moment. If we offer a new token set, it will be Lara.
